migrate.py

```diff
@@ -2098,6 +2116,11 @@ def main():
 
         logger.info('Starting the migration...')
 
+        # we need to be able to import collections when evaluating filters and tests
+        loader = AnsibleCollectionLoader()
+        loader._n_configured_paths = [os.path.join(args.vardir, 'collections')]
+        sys.meta_path.insert(0, loader)
```

Comment:
this probably deserves a log message:
```suggestion
        sys.meta_path.insert(0, loader)
        logger.info('Successfully injected AnsibleCollectionLoader into the import machinery...')
```